validation-test/Sema/type_checker_perf_failing/generic_conformance.swift.gyb

```diff
@@ -0,0 +1,20 @@
+// RUN: %scale-test --begin 1 --end 5 --step 1 --select incrementScopeCounter %s --polynomial-threshold=2.1
```

Comment:
Passing. The scale counter here could use a little improvement but I haven't come up with something better at this point.